src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide.mdx

```diff
@@ -9,184 +9,199 @@ redirects:
+4. Once all legacy runtime monitors on the private location have been updated, you can uninstall your containerized private minion (CPM). 
 
-When you install the job manager, we recommend that you use the same private location key as your existing CPM. This simplifies the migration process. Keep in mind that:
+Keep in mind that:
 
 * CPMs execute monitors that use the legacy runtime
 * Job managers execute monitors that use the newest runtime
 * Ping monitors are load balanced, so both CPM and job managers can execute them
 
-1. Retrieve your existing private location's key by [editing the private location](https://one.newrelic.com/synthetics/private-location-list)
-2. [Install job manager](/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager) using this key.
-3. The [runtime upgrades UI](/docs/synthetics/synthetic-monitoring/using-monitors/runtime-upgrade-ui/) will start testing the legacy runtime monitors assigned to this private location and populating test results. Running these tests and populating the UI may take a few hours. Use the UI to upgrade monitor runtimes. 
-4. Once all legacy runtime monitors on the private location have been updated, you can uninstall your containerized private minion (CPM). 
+## Migrate synthetics running in Docker [#synthetics-docker]
 
-If you're running more than one Docker private location container in the same host, you'll have port and other resource conflicts. To avoid this port contention, make sure you do the following when you start setting up job managers:
+If you're running more than one private location on a Docker container in the same host, you'll have port and other resource conflicts. To avoid this port conflict, make sure that you either:
 
 * Run job managers and CPMs on different hosts.
 * Run each job manager on a separate host.
 * Run each CPM on a different host.
 
-Want a fresh start instead? Create a new private location for the job manager. Your private location configuration and migration path comes down to your own personal preference. Using a new private location will prevent the runtime upgrades UI from proactively testing your legacy runtime monitors. This path would also require you to modify monitor configurations to use the new private location.
+## Create a new private location  for job manager [#create-new]
+
+If you want a fresh start, then you can [create a new private location](/docs/synthetics/synthetic-monitoring/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations/#create-location) for the job manager. 
+
+Creating a new private location prevents the runtime upgrades UI from proactively testing your legacy runtime monitors. This path would also require you to modify monitor configurations to use the new private location.
 
 ## Upgrade your monitors [#upgrade]
```

Comment:
Can you differentiate "Upgrading your monitors" and "Migrating from CPM to job manager"? Can these sections be merged into one? When we say "Upgrade your monitors", do we mean "upgrade your CPMs" instead?

Reply:
Installing Job Manager allows you to upgrade the runtime of monitors using private locations. Both steps should be included in a single doc so that customers don't lose track of the steps needed to complete the migration.

Going further, the four steps called out in #monitorMigration are the steps required. You need to obtain a private location key (we recommend using the existing private location), install Job Manager, upgrade your monitors to the latest runtimes so that they start running on Job Manager, then CPM can be uninstalled.